### ISSUE TITLE
Room picture generation never finish on the webpage but finished in the replicate?

### ISSUE THREAD

**Issue by mmtechhk** [Nutlope/roomGPT#69]
Thanks for the support. I have successfully clone the project, fix the DB connection issue, google login issue, and the tab issue (by comment the following line of code in[ dream.tsx], tags: [data?.remainingGenerations > 3 ? "paid" : "free"],)

**However, after logging in and updating my photo, I haven't seen any changes. I checked the replication status and it's finished, but the webpage keeps hanging and loading. Can anyone help me with this final step?**

URL: https://room.mic9chan.com

Keep waiting on this page:
![image](https://user-images.githubusercontent.com/86524726/231721931-510eacdd-069b-496a-b5d0-edbc3dba5281.png)

Replicate resule is good:
![image](https://user-images.githubusercontent.com/86524726/231722184-9fbc8ab2-4eff-4123-994e-df5ba8c85c04.png)

**Comment by mmtechhk** [comment]
@R3P-tech You have the solution or not?

**Comment by Nutlope** [comment]
Hey all. I've just simplified the repo to remove auth and payments so you can clone and deploy very easily, only with a Replicate environment variable. Try it out and if you still have any problems, please open a new issue and I will take a look!
